renku/api/repository.py

```diff
@@ -132,10 +133,9 @@ def cwl_prefix(self):
         return str(self.workflow_path.resolve().relative_to(self.path))
 
     @cached_property
```

Comment:
Since we are not creating new attrs object (no additional initializers), I think there is no need to have it cached. In fact caching it might cause problems if I manually change the attributes on the project (e.g. path).

Reply:
ah true, I removed the instantiation of the object here